frontend/packages/knative-plugin/src/utils/__mocks__/traffic-splitting-utils-mock.ts

```diff
@@ -1,87 +1,60 @@
-import { RevisionModel, ServiceModel } from '../../models';
+import * as _ from 'lodash';
+import {
+  MockKnativeResources,
+  revisionObj,
+} from '@console/dev-console/src/components/topology/__tests__/topology-knative-test-data';
 import { RevisionKind, ServiceKind as knativeServiceKind } from '../../types';
 
-export const mockTrafficData = [
-  { percent: 50, tag: 'tag-1', revisionName: 'rev-1' },
-  { percent: 50, tag: 'tag-2', revisionName: 'rev-2' },
+export const mockServiceData: knativeServiceKind = _.cloneDeep(
+  MockKnativeResources.ksservices.data[0],
```

Comment:
can  directly import `knativeServiceObj` as you did for revisionObj

Reply:
done